receiver/prometheusreceiver/internal/transaction_test.go

```diff
@@ -197,14 +271,63 @@ func nopObsRecv() *obsreport.Receiver {
 	})
 }
 
+func getLabel(name string, value string) labels.Label {
```

Comment:
Why having this instead of using the normal `labels.Label{Name:  name, Value: value}`?

Reply:
Addressed